--- libmpcodecs/ad_spdif.c	2024-11-25 19:52:49.000000000 +0700
+++ libmpcodecs/ad_spdif.c.new	2025-08-18 08:16:35.745522564 +0700
@@ -54,7 +54,7 @@
     return 0;
 }
 
-static int write_packet(void *p, const uint8_t *buf, int buf_size)
+static int write_packet(void *p, unsigned char *buf, int buf_size)
 {
     int len;
     struct spdifContext *ctx = p;

--- libmpdemux/muxer_lavf.c	2024-11-25 19:52:50.000000000 +0700
+++ libmpdemux/muxer_lavf.c.new	2025-08-18 08:13:22.501231468 +0700
@@ -77,7 +77,7 @@
 	{NULL, NULL, 0, 0, 0, 0, NULL}
 };
 
-static int mp_write(void *opaque, const uint8_t *buf, int size)
+static int mp_write(void *opaque, unsigned char *buf, int size)
 {
 	muxer_t *muxer = opaque;
 	return stream_write_buffer(muxer->stream, buf, size);
